guides/micronaut-microservices-services-discover-eureka/bookrecommendation/groovy/src/test/groovy/example/micronaut/BookControllerSpec.groovy

```diff
@@ -1,31 +1,31 @@
 package example.micronaut
 
-import io.micronaut.core.type.Argument
 import io.micronaut.http.HttpRequest
-import io.micronaut.http.HttpResponse
-import io.micronaut.http.HttpStatus
-import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.StreamingHttpClient
 import io.micronaut.http.client.annotation.Client
 import io.micronaut.test.extensions.spock.annotation.MicronautTest
-import spock.lang.Specification
 import jakarta.inject.Inject
+import reactor.core.publisher.Flux
 import spock.lang.IgnoreIf
+import spock.lang.Specification
 
 @MicronautTest
 class BookControllerSpec extends Specification {
 
     @Inject
     @Client("/")
-    HttpClient client
+    StreamingHttpClient client
```

Comment:
why do we want to use a `StreamingHttpClient` vs a `HttpClient` for this?

Reply:
same as in micronaut-microservices-services-discover-consul, to make the Groovy test consistent with the Java and Kotlin